### PR TITLE
dns/dnscrypt-proxy: add blacklist support

### LINKED ISSUE #1183: [Feature Request] DNSCrypt - Blacklist Support
DNSCrypt-Proxy has built-in ability to use domain blacklists:
https://github.com/jedisct1/dnscrypt-proxy/wiki/Public-blacklists

Are there any plans to support this (and possible logging of blocked requests) from the within the existing plug-in?

### PR DESCRIPTION
**Pull request by mimugmail**
Closes #1183 

I copied the BIND DNSBL functionality to dnscrypt-proxy.

This would avoid triple forwards when someone needs Unbound for DNS, BIND for DNSBL and dnscrypt for privacy. 

As dnscrypt has excellent overrides / forwarding, in most cases also Unbound would not be mandatory now.

### CODE REVIEW

File: dns/dnscrypt-proxy/pkg-descr [hunk 1]
```diff
@@ -16,6 +20,10 @@ Plugin Changelog
 1.0
 
 * Initial release
```
Comment (fichtner) [inline dns/dnscrypt-proxy/pkg-descr]: Can remove this line for sure :)

Comment (mimugmail) [reply dns/dnscrypt-proxy/pkg-descr]: Done :)

**Comment by mimugmail** [comment]
Thanks for review guys <3

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**